### PR TITLE
Add autosuggest for about and mentions.

### PR DESCRIPTION
**Pull request by khatton-ft**
Non-classification types (i.e. people, organisations, locations and
topics) are included in the search through the ES mapping file.
Also added an option (which defaults to false) to output trace logging
from the ES client, which shows exactly what HTTP requests and responses
are being made.

### CODE REVIEW

**Comment by peteclark-ft** [comment]
Can you update the api.yml too - this changes the contract as type is no longer a required field.

File: resources/handler.go [hunk 1]
```diff
@@ -34,8 +34,12 @@ func (h *Handler) ConceptSearch(w http.ResponseWriter, req *http.Request) {
 	}
 
 	if isAutocomplete {
-		if foundQ && foundConceptType {
-			concepts, searchErr = h.service.SuggestConceptByTextAndType(q, conceptType)
+		if foundQ {
```
Comment (peteclark-ft) [inline resources/handler.go]: This means that leaving out the type field implies a query for abouts/mentions types only? I guess that's fine for J30, but could that be confusing long term?

Comment (khatton-ft) [inline resources/handler.go]: As I understand it, that is the requirement. "Classification" types  (https://www.lucidchart.com/publicSegments/view/7cd85c86-4d73-408d-9e57-3219f5f99e28/image.png) should be excluded from the abouts/mentions annotations.

File: main.go [hunk 2]
```diff
@@ -71,6 +71,12 @@ func main() {
 		Desc:   "The maximum number of autocomplete results returned",
 		EnvVar: "AUTOCOMPLETE_LIMIT",
 	})
+	esTraceLogging := app.Bool(cli.BoolOpt{
```
Comment (izzyblues) [inline main.go]: <3

**Pull request merged**